documentation/reference/plugins/jwt-keycloak.md

```diff
@@ -1,42 +1,10 @@
 # JWT Keycloak
 
-## Example
+## Configuration reference
 
-```yaml
-services:
-  - name: MY_REST_API
-    tags: [_NS_]
-    plugins:
-      - name: jwt-keycloak
-        tags: [_NS_]
-        enabled: true
-        config:
-          allowed_iss:
-            - https://keycloak/auth/realms/REALM
-          allowed_aud: an-audience-ref
-          #access_token_header: Authorization
-          #realm: kong
-          #disable_access_token_header: false
-          #run_on_preflight: true
-          #iss_key_grace_period: 10
-          #maximum_expiration: 0
-          #claims_to_verify:
-          #- exp
-          #algorithm: RS256
-          #well_known_template: %s/.well-known/openid-configuration
-          #cookie_names: []
-          #scope: null
-          #realm_roles: null
-          uri_param_names: []
-          client_roles: null
-          anonymous: null
-          consumer_match: true
-          #consumer_match_claim: azp
-          #consumer_match_ignore_not_found: false
-          #consumer_match_claim_custom_id: false
-```
+This is a custom plugin managed by the API Program Services team.
 
-## Key Fields
+Here is a list of the parameters which can be used in this plugin's `config` section:
 
 | Field                       | Type     | Default | Description                                                                                                                                         |
```

Comment:
thoughts on this table style vs what is in the [template](https://dev.developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/reference/!plugin-template/#configuration-reference) (modelled after Kong docs)?

Reply:
I think the table is a bit cleaner

Reply:
OK I'm good with a table but I would like to keep all the 'columns' in the template. What I like with the template style is that 'columns' are optional, so the line doesn't _need_ to be super long. 

Whereas the table will sort of need to be long. Or columns can be removed if not applicable to the entire table